db/schema.rb

```diff
@@ -361,4 +362,4 @@
   add_foreign_key "oauth_access_grants", "oauth_applications", column: "application_id"
   add_foreign_key "oauth_access_tokens", "oauth_applications", column: "application_id"
   add_foreign_key "users", "oauth_applications", column: "source_application_id"
-end
+end
```

Comment:
Please add a newline at the end of the line. It might be a setting in your editor btw; I know there is one for VSCode and — if I remember correctly — for Sublime text as well, to do automatically.

Reply:
Why should there be a new line?

Reply:
It's a best practice in Ruby per the Ruby style guide. https://rubystyle.guide/#newline-eof

Reply:
New line added and I configured Rubymine to add it in the future.